src/I18n/TranslatorRegistry.php

```diff
@@ -87,7 +87,7 @@ class TranslatorRegistry
      * A CacheEngine object that is used to remember translator across
      * requests.
      *
-     * @var (\Psr\SimpleCache\CacheInterface&\Cake\Cache\CacheEngineInterface)|null
+     * @var (\Psr\SimpleCache\CacheInterface & \Cake\Cache\CacheEngineInterface)|null
```

Comment:
Looks like have to remove the spacing from intersection types. However, the space does make it a lot easier to read.